### PR TITLE
Fix required xpaths for the selection of the model server template

### PR DESCRIPTION
**Pull request by jstourac**
This change is necessary for the latest RHOAI 2.9 preRC build.

---

Local check:

```
./ods_ci/run_robot_test.sh --extra-robot-args '-i ODS-2626 -i ODS-2519 -i ODS-2552 -i ODS-2555 -i ODS-2268' --open-report true
```

CI: `rhods-smoke/5113/`

### CODE REVIEW

**Comment by manosnoam** [comment]
To verify KW `Create Model Server` please run suite `420__model_serving.robot`.
In addition, consider adding it directly into `Serve Model` since `Create Model Server` is always called before `Serve Model` (except in test `Verify GPU Model Deployment Via UI`, but it seems OK to have that there too).

File: ods_ci/tests/Resources/Page/ODH/ODHDashboard/ODHModelServing.resource [hunk 1]
```diff
@@ -411,12 +411,12 @@ Deploy Kserve Model Via UI  #robocop: disable
     Click Button    Deploy
     Wait Until Page Does Not Contain Element    xpath=${KSERVE_MODAL_HEADER}   timeout=60s
 
-Select Kserve Serving Runtime
+Set Model Server Runtime
     [Documentation]    Opens the Runtime dropdown in the deploy modal for kserve models and select the given runtime
```
Comment (FedeAlonso) [inline ods_ci/tests/Resources/Page/ODH/ODHDashboard/ODHModelServing.resource:415]: The doc says "...for **kserve** models...". Shouldn't it be more generic (like in the title, which now it's not about KServe)

**Review by FedeAlonso — CHANGES_REQUESTED** [review]

**Comment by jstourac** [comment]
> To verify KW Create Model Server please run suite 420__model_serving.robot.

That is the reason I run the ODS-2268 - is it really necessary to run the whole suite for this change? :thinking: 

> In addition, consider adding it directly into Serve Model since Create Model Server is always called before Serve Model (except in test Verify GPU Model Deployment Via UI, but it seems OK to have that there too).

I'm not gonna do that - I'm not gonna touch the code more for this change now. I'm not the expert here and this was supposed to be just an xpath change to work with the current build. If we want more changes, I am okay to hand this over to somebody who is more experienced here and knows the code.

File: ods_ci/tests/Resources/Page/ODH/ODHDashboard/ODHModelServing.resource (hunk 1 above)
Comment (jstourac) [reply ods_ci/tests/Resources/Page/ODH/ODHDashboard/ODHModelServing.resource]: updated

**Review by FedeAlonso — APPROVED** [review]

**Review by manosnoam — APPROVED** [review]

**Pull request merged**